src/Windows/Avalonia.Win32/DirectX/DxgiRenderTarget.cs

```diff
@@ -24,6 +24,14 @@ internal unsafe class DxgiRenderTarget : EglPlatformSurfaceRenderTargetBase
         private IUnknown? _renderTexture;
         private RECT _clientRect;
 
+        [Flags]
+        enum AssociationFlags : ushort
```

Comment:
Please define this in `DirectXEnums.cs` for consistency (the name could be `DXGI_MWA`).
Also, the second parameter for `MakeWindowAssociation()` in `DirectX.idl` could probably now be typed.

Reply:
I will do it asap

Reply:
@MrJul  done